hack/run-all.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Run all the important scripts in a sensible order,
+# handy for switching branches, etc
+wd="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+$wd/clean-deps.sh &&
+$wd/install-deps.sh &&
+echo '[build-assets]' &&
+grunt build &&
+echo '[test-assets]' &&
+grunt test &&
+grunt test-integration
+# $wd/test-integration-headless.sh &&
+# $wd/verify-dist.sh
```

Comment:
correct, its always going to fail if you haven't committed your dist yet, so if you are running through the clean/install/build/test you may or may not have actually commited dist already.  Plus with us having enabled travis now you'll know pretty quick if you forgot that step